core/domain/image_validation_services.py

```diff
@@ -48,7 +48,10 @@ def validate_image_and_filename(
         ValidationError. Image or filename supplied fails one of the
             validation checks.
     """
-    if entity_type == feconf.ENTITY_TYPE_BLOG_POST:
+    if (
+        entity_type == feconf.ENTITY_TYPE_BLOG_POST or
+        feconf.ENTITY_TYPE_CLASSROOM
```

Comment:
Wait wait, what? Maybe entity_type in [BLOG_POST, CLASSROOM]?

Reply:
Done.